### PR TITLE
Added pictures of cards and made sure tests pass.

### PR DESCRIPTION
**Pull request by barrieca**
I've added pictures for the Spanish deck cards. I've also made sure that all unit tests in the model have passed.

### CODE REVIEW

File: src/test/java/models/testGame.java [hunk 1]
```diff
@@ -62,6 +62,8 @@ public void testRemoveFunction(){
         g.remove(2);
         assertEquals(0,g.cols.get(2).size());
         assertEquals(1,g.getScore());
+        g.remove(2);
+        assertEquals(0,g.cols.get(2).size());
```
Comment (parkj6) [inline src/test/java/models/testGame.java:66]: Whats up with the duplicate here?

**Comment by parkj6** [comment]
Where did you get the spanish deck? does it align with the card name? (also takes care of #20)

File: src/test/java/models/testGame.java (hunk 1 above)
Comment (barrieca) [inline src/test/java/models/testGame.java:66]: That tests that the deck is empty, which was not tested before.

**Comment by barrieca** [comment]
This is just temporary. I have more code coming soon.

Working on getting the cards to work with the game.

**Comment by barrieca** [comment]
Working on getting the cards to work with the game.

**Comment by parkj6** [comment]
I'm just gonna merge pull request so I can download the Spanish deck pictures.

**Pull request closed without merging**